conjure-lib/src/main/java/com/palantir/conjure/java/lib/internal/ConjureIntegerList.java

```diff
@@ -55,6 +55,10 @@ public boolean addAll(int index, Collection<? extends Integer> collection) {
         return delegate.addAllAtIndex(index, target);
     }
 
+    public void addAll(int... source) {
```

Comment:
here as well

Reply:
in practice perhaps these should take `int[]`/etc for clarity? If we're not careful with varargs, we can end up allocating (small) arrays unnecessarily